Scribe/Components/ParentTableViewCell/ParentTableViewCell.swift

```diff
@@ -68,4 +70,103 @@ extension ParentTableViewCell: UITableViewDataSource {
+  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    print(data?.section[indexPath.row].sectionTitle ?? "")
+    
+    if let section = data?.section[indexPath.row] {
+      switch section.sectionState {
+      case .github:
+        openURLString(urlString: "https://github.com/scribe-org/Scribe-iOS", withEncoding: false)
+      case .matrix:
+        openURLString(urlString: "https://matrix.to/#/#scribe_community:matrix.org", withEncoding: true)
+      case .wikimedia:
+        // Push a new screen
+        print("Details about Wikimedia and Scribe")
+      case .shareScribe:
+        showShareSheet()
+      case .rateScribe:
+        showRateScribeUI()
+      case .bugReport:
+        openURLString(urlString: "https://github.com/scribe-org/Scribe-iOS/issues", withEncoding: false)
+      case .email:
+        showEmailUI()
+//      case .appHints:
+//        // reset functionality
+//        print("Resets app hints")
+      case .privacyPolicy:
+        // Push a new screen
+        print("Scribe privacy policy page")
+      case .licenses:
+        // Push a new screen
+        print("Licenses page")
+      case .appLang: break
+      case .specificLang: break
+      }
+    }
+    
+    if let selectedIndexPath = tableView.indexPathForSelectedRow {
+      tableView.deselectRow(at: selectedIndexPath, animated: false)
+    }
+  }
+  
+  func openURLString(urlString: String, withEncoding: Bool) {
+    if withEncoding {
+      let encodedString = urlString.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed)
+      guard let encodedURLString = encodedString, let url = URL(string: encodedURLString) else { return }
+      UIApplication.shared.open(url, options: [:], completionHandler: nil)
+    } else {
+      guard let url = URL(string: urlString) else { return }
+      UIApplication.shared.open(url, options: [:], completionHandler: nil)
+    }
+  }
+  
+  func showRateScribeUI() {
+    if #available(iOS 14.0, *) {
+      guard let scene = UIApplication.shared.foregroundActiveScene else { return }
+      SKStoreReviewController.requestReview(in: scene)
+    } else {
+      let alert = UIAlertController(title: "Enjoying Scribe?", message: "Rate Scribe on the App Store.", preferredStyle: .alert)
+      alert.addAction(UIAlertAction(title: "Continue", style: .default, handler: openScribeAppStore(alert:)))
+      alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
+      parentViewController?.present(alert, animated: true)
+    }
+  }
+  
+  func openScribeAppStore(alert: UIAlertAction) {
+    openURLString(urlString: "itms-apps: //itunes.apple.com/app/id1596613886", withEncoding: true)
+  }
+  
+  func showEmailUI() {
+    if MFMailComposeViewController.canSendMail() {
+      let mailComposeViewController = MFMailComposeViewController()
+      mailComposeViewController.mailComposeDelegate = self
+      mailComposeViewController.setToRecipients(["scribe.language@gmail.com"])
```

Comment:
Hey! :wave: 

I noticed that it's possible to set the subject line and the message body as well like the recipients here. I'm thinking that the message body is likely fine staying blank, but would it make sense perhaps to have some default set for the subject line? Just more so thinking in terms of what would be easier or expected from the user perspective.

Reply:
Hey Will!

I did consider that. But we have separate sections for almost all the reasons a user might want to approach us. For example, to report a bug there is a section and to rate and share feedback there's the rating section. So writing an email seemed like a very ambiguous use case. That's why I didn't add a subject line in case the user wished to write an email for a completely different reason. Could be a complaint but could also be an email of appreciation 😅

Reply:
I agree that the use case for emails is going to ambiguous 🤔 I just checked and we're still at a grand total of `0` emails 😄 With that being said, maybe something inviting to the user? `Hey Scribe!`, for instance?

Reply:
Sounds good @andrewtavis! We can do that. Just a tiny problem. I kind of committed my changes for the privacy policy issue (#317) to the branch already. So if I push it now that'll be added to the PR too. So would it be okay if I made this change in the next PR? I am so sorry! 🙈

Reply:
100% ok, @SaurabhJamadagni 😊 No need to apologize :)